pkg/alicloud/client/client.go

```diff
@@ -222,6 +224,60 @@ func (c *ecsClient) ShareImageToAccount(ctx context.Context, regionID, imageID,
 	return err
 }
 
+// DetachECSInstancesFromSshKey finds all ECS instances and detach them from the specified SSH key pair.
+func (c *ecsClient) DetachECSInstancesFromSshKey(keyName string) error {
```

Comment:
Currently this func is not a trivial one and there are no unit tests related to it. I don't know what kind of possibilities the alicloud sdk has for mocking/faking but it would be nice to have unit tests for this function, if possible.

Reply:
Good suggestion. Unfortunately, aliyunclient only expose a struct type. I would not introduce complicated reflection techniques for to implement this UT.